test/tests/functional/pbs_cgroups_hook.py

```diff
@@ -501,17 +485,25 @@ def set_vntype(self, typestring='myvntype'):
         """
         pbs_home = self.server.pbs_conf['PBS_HOME']
         vntype_file = os.path.join(pbs_home, 'mom_priv', 'vntype')
+        self.logger.info("Setting vntype to %s in %s" %
+                         (typestring, vntype_file))
+        cmd = ['/bin/echo', '%s' % typestring]
+        self.logger.info("Command is: %s" % (cmd))
         with open(vntype_file, "w") as fd:
-            fd.write(typestring)
+            ret = self.du.run_cmd(self.mom.hostname, cmd, stdout=fd, sudo=True)
+        if ret['rc'] != 0:
+            self.skipTest("pbs_cgroups_hook: need root privileges")
 
     def remove_vntype(self):
         """
         Unset the vnode type for the local mom.
         """
         pbs_home = self.server.pbs_conf['PBS_HOME']
         vntype_file = os.path.join(pbs_home, 'mom_priv', 'vntype')
-        if os.path.isfile(vntype_file):
-            os.remove(vntype_file)
+        cmd = ['/bin/rm', '-f', '%s' % vntype_file]
+        ret = self.du.run_cmd(self.mom.hostname, cmd, sudo=True)
```

Comment:
I looked at the existing sudo rules and it should be supported if not then we need to add a new rule.

Reply:
Done.